content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-ha.md

```diff
@@ -193,30 +211,30 @@ apt-mark hold kubelet kubectl && \
 systemctl restart kubelet
 ```
 
-In this example a _deb_-based system is assumed and `apt-get` is used for installing the upgraded software. On _rpm_-based systems it will be `yum install <PACKAGE>=<NEW-K8S-VERSION>` for all packages.
+In this example a _deb_-based system is assumed and `apt-get` is used for installing the upgraded software. On rpm-based systems the command is `yum install <PACKAGE>=<NEW-K8S-VERSION>` for all packages.
 
-Now the new version of the `kubelet` should be running on the host. Verify this using the following command on the respective host:
+Verify that the new version of the kubelet is running:
 
 ```shell
 systemctl status kubelet
 ```
 
-Verify that the upgraded node is available again by executing the following from wherever you run `kubectl` commands:
+Verify that the upgraded node is available again by running the following command from wherever you run `kubectl`:
 
 ```shell
 kubectl get nodes
 ```
 
-If the `STATUS` column of the above command shows `Ready` for the upgraded host, you can continue (you may have to repeat this for a couple of time before the node gets `Ready`).
+If the `STATUS` column shows `Ready` for the upgraded host, you can continue. You might need to repeat the command until the node shows `Ready`.
 
 ## If something goes wrong
 
-If the upgrade fails the situation afterwards depends on the phase in which things went wrong:
+If the upgrade fails, see whether one of the following scenarios applies:
 
-1. If `/tmp/kubeadm upgrade apply` failed to upgrade the cluster it will try to perform a rollback. Hence if that happened on the first master, chances are pretty good that the cluster is still intact.
+- If `/tmp/kubeadm upgrade apply` failed to upgrade the cluster, it will try to perform a rollback. If this is the case on the first master, the cluster is probably still intact.
```

Comment:
We should be able to replace any /tmp/kubeadm references with just kubeadm, since we are upgrading kubeadm using the package manager again.